kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -163,15 +161,15 @@ private PartitionedTableId getRecordTable(SinkRecord record) {
 
   /**
    * Create the table which doesn't exist in BigQuery for a (record's) topic when autoCreateTables config is set to true.
-   * @param record Kafka Sink Record to be streamed into BigQuery.
+   * @param topic Kafka Sink Record topic.
    * @param baseTableId BaseTableId in BigQuery.
    */
-  private void maybeCreateTable(SinkRecord record, TableId baseTableId) {
+  private void maybeCreateTable(TableId baseTableId, String topic) {
```

Comment:
Not sure about the naming of this. Maybe something like `tryCreateTable` or `createIfRequired`?

Reply:
Make sense, modified the name to be `createTableIfNecessary`.